aoa-jupyter-lab

```diff
@@ -147,13 +144,35 @@ RUN echo "${NB_USER} ALL=(ALL) NOPASSWD: /usr/bin/java" >> /etc/sudoers
 #ENV REQUESTS_CA_BUNDLE=/opt/conda/ssl/cert.pem
 
 # Add AOA CLI config file
-RUN mkdir -p /home/jovyan/.aoa && \
-    echo 'aoa_credentials: YWRtaW46YWRtaW4=' >> /home/jovyan/.aoa/config.yaml && \
-    echo 'aoa_url: http://aoa-core:8080' >> /home/jovyan/.aoa/config.yaml   && \
-    echo 'auth_mode: basic' >> /home/jovyan/.aoa/config.yaml
+RUN mkdir -p /home/${NB_USER}/.aoa && \
+    echo 'aoa_credentials: YWRtaW46YWRtaW4=' >> /home/${NB_USER}/.aoa/config.yaml && \
+    echo 'aoa_url: http://aoa-core:8080' >> /home/${NB_USER}/.aoa/config.yaml   && \
+    echo 'auth_mode: basic' >> /home/${NB_USER}/.aoa/config.yaml
+
+# Fix permissions
+RUN fix-permissions "${CONDA_DIR}/ssl" && fix-permissions "/home/${NB_USER}/.aoa" && fix-permissions "/teradata" && chown -R ${NB_USER} "/home/${NB_USER}/work"
+
+# Remove default pyhon3 kernel
+RUN jupyter kernelspec uninstall -y python3
+
+# Create python 3.6 kernel
+RUN conda create -n py36 python=3.6 ipykernel
+RUN conda run -n py36 pip uninstall -y jupyter_client
+RUN conda run -n py36 pip install jupyter_client
+RUN conda run -n py36 python -m ipykernel install --name=python3 --display-name "Python 3.6 (ipykernel)"
+
+# Create python 3.7 kernel
+RUN conda create -n py37 python=3.7 ipykernel
+RUN conda run -n py37 pip uninstall -y jupyter_client
+RUN conda run -n py37 pip install jupyter_client
+RUN conda run -n py37 python -m ipykernel install --name=py37 --display-name "Python 3.7 (ipykernel)"
 
 # Fix permissions
-RUN fix-permissions "${CONDA_DIR}/ssl" && fix-permissions "/home/${NB_USER}/.aoa" && fix-permissions "/teradata"
+RUN fix-permissions "/home/${NB_USER}/.local"
+
+# Set up Jupyter resource usage
+ENV MEM_LIMIT=1073741824
+ENV CPU_LIMIT=1
```

Comment:
what does this do?

Reply:
This is a soft limit in the image, we can set the hard limit through the jupyterhub config, but in both cases it shows the mem usage and limit. For cpu only works the hard limit, and nothing shows on the ui. 
https://github.com/jupyter-server/jupyter-resource-usage#memory-usage-warning-threshold